### PR TITLE
Add HoTT to Windows Installer (master)

### PR DESCRIPTION
**Pull request by MSoegtropIMC**
This is the same as #10588 just for master.

### CODE REVIEW

**Comment by MSoegtropIMC** [comment]
This has some unexpected effects and shouldn't be merged yet.
See `https://github.com/HoTT/HoTT/issues/1064`

File: dev/build/windows/makecoq_mingw.sh [hunk 1]
```diff
@@ -1971,6 +1971,37 @@ function make_addon_gappa {
   fi
 }
 
+# HoTT Homotopy Type Theory
+
+function make_addon_hott {
+  if build_prep_overlay HoTT ; then
+    installer_addon_section hott "HoTT" "Homotopy Type Theory" ""
+    logn autogen ./autogen.sh
```
Comment (SkySkimmer) [inline dev/build/windows/makecoq_mingw.sh]: You should maybe use -skip-submodules to avoid wasting time/bandwidth cloning coq

Comment (MSoegtropIMC) [reply dev/build/windows/makecoq_mingw.sh]: In which step are the sub modules cloned? Does autogen or configure do this? `build_prep_overlay HoTT` just downloads a `.tar.gz` file from the  git repo server.

**Comment by SkySkimmer** [comment]
autogen does it, but only when .git exists so if you're downloading a 
tarball submodules aren't cloned and my suggestion is redundant

Gaëtan Gilbert

**Pull request closed without merging**